### PR TITLE
SDSS-1302: Adding additional layout with widths baked in.

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Added two additional layouts: Two Column Larger Left and Two Column Larger Right
- **Note**: I did not add another for Equal Columns because it made sense to use the existing 2 columns and add the word equal to the label. I think this means we don't have to account for pages already using the 2 column.

# Review By (Date)
- As time allows

# Criticality
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to a basic page (try other CTs too)
3.   Verify you see the additional templates.
4. Add them and add content
5. Try out background colors.
6. Try the two layouts to make sure the templates work as expected.
7. Verify...

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SDSS-1302](https://stanfordits.atlassian.net/browse/SDSS-1302)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by joegl — CHANGES_REQUESTED** [review]
Everything looks good just a few small questions/requests. Tested it in Gitpod and works great 👍

File: docroot/profiles/sdss/sdss_profile/config/sync/paragraphs.paragraphs_type.stanford_layout.yml [hunk 1]
```diff
@@ -17,5 +17,7 @@ behavior_plugins:
     available_layouts:
       layout_paragraphs_sdss_1_column: '1 Column'
       layout_paragraphs_sdss_2_column: '2 Column'
```
[the remark this answers is not in the document]
Comment (jenbreese) [reply docroot/profiles/sdss/sdss_profile/config/sync/paragraphs.paragraphs_type.stanford_layout.yml:19]: @joegl I wasn't sure about that. Will it cause problems for pages using the existing 50/50 layout?

[the remark this answers is not in the document]
Comment (joegl) [reply docroot/profiles/sdss/sdss_profile/config/sync/paragraphs.paragraphs_type.stanford_layout.yml:19]: As long as we leave the machine name as `layout_paragraphs_sdss_2_column` and only change the 2 column label it should be fine.

[the remark this answers is not in the document]
Comment (jenbreese) [reply docroot/profiles/sdss/sdss_profile/config/sync/paragraphs.paragraphs_type.stanford_layout.yml]: Hey @joegl Ok. This is the last comment to fix in this PR. I'll do this next.

[the remark this answers is not in the document]
Comment (joegl) [reply docroot/profiles/sdss/sdss_profile/config/sync/paragraphs.paragraphs_type.stanford_layout.yml]: Awesome, thanks Jen! 👍

[the remark this answers is not in the document]
Comment (jenbreese) [reply docroot/profiles/sdss/sdss_profile/config/sync/paragraphs.paragraphs_type.stanford_layout.yml]: Hi @joegl I double checked and this PR is ready for you to review the naming change requested above.

**Review by joegl — APPROVED** [review]
LGTM! Thanks Jen 👍

**Pull request merged**